generics-sop/src/Generics/SOP/Instances.hs

```diff
@@ -206,7 +206,9 @@ deriveGeneric ''Data.Semigroup.Max -- new
 deriveGeneric ''Data.Semigroup.First -- new
 deriveGeneric ''Data.Semigroup.Last -- new
 deriveGeneric ''Data.Semigroup.WrappedMonoid -- new
+#if __GLASGOW_HASKELL__ <= 901
```

Comment:
`!MIN_VERSION_base(4,16,0)`

Reply:
I did try that, maybe I didn't completely understand the MIN piece... it didn't make sense at the time.